code/style.css

```diff
@@ -1,7 +1,188 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  height: 100%;
+  background-color: blanchedalmond;
+  font-family:'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  color:rgb(53, 48, 48);
+  margin: auto;
+  padding: 5%;
+}
+
+.wrapper {
+  position: relative; 
+}
+
+ul {
+  list-style: none;
+  text-align: right;
+  padding: 20px;
+  background-color: darkseagreen;
+  color: white;
+  margin: 0;
+}
+
+li {
+  display: inline;
+  padding: 10px;
+}
+
+li a{
+  font-size: large;
+  color: blanchedalmond;
+}
+
+#hero-image {
+  height: 500px;
+  background-image: url("pexels-surene-palvie-2062061.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  overflow-x: hidden;
+  flex-direction: column;
+}
```

Comment:
What does "position: relative;" do here? tried to uncheck it in inspect but it doesnt really do anything, dont think 
overflow-x :hidden; does anything either I have seen some position: relative in your code but dont really see what they do, will mark them up so you can see where they are